regions_www/m/server.R

```diff
@@ -82,6 +85,23 @@ lsoa_legend_df <- data.frame(
 # # # # # # # #
 shinyServer(function(input, output, session) {
 
+  # Save current region in a state variable
+  onBookmark(function(state) {
+    state$values$current_region <- region$current
+  })
+
+  # Restore state's region into a region variable
+  onRestore(function(state) {
+    region$state_region <<- state$values$current_region
+
+    region$state_lng <<- state$input$map_center$lng
+    region$state_lat <<-  state$input$map_center$lat
+    region$state_mzoom <<- state$input$map_zoom
```

Comment:
Do you need `<<-`? I know we use them but we need to stop.

Reply:
Good point. No, I don't need it. Have now removed it. At some point, we should rid of all of them.